### PR TITLE
HYP-297: Halborn Fixes

### PR DESCRIPTION
**Pull request by ankurdubey521**

### CODE REVIEW

File: contracts/hyphen/interfaces/ILPToken.sol [hunk 1]
```diff
@@ -78,16 +78,14 @@ interface ILPToken {
 
     function totalSupply() external view returns (uint256);
 
-    function transferFrom(
+    function abitransferFrom(
```
Comment (tomarsachin2271) [inline contracts/hyphen/interfaces/ILPToken.sol:81]: Why the name is changed?

File: test/LiquidityPool.tests.ts [hunk 2]
```diff
@@ -341,6 +337,7 @@ describe("LiquidityPoolTests", function () {
             let toChainId = 1;
 
             await addTokenLiquidity(tokenAddress, liquidityToBeAdded, owner);
+            await executorManager.addExecutor(executor.address);
```
Comment (tomarsachin2271) [inline test/LiquidityPool.tests.ts:340]: Why is this needed in this function?

File: test/LiquidityPool.tests.ts [hunk 3]
```diff
@@ -371,6 +368,7 @@ describe("LiquidityPoolTests", function () {
...
```
Comment (tomarsachin2271) [inline test/LiquidityPool.tests.ts:371]: Why is this needed in this function?

File: test/LiquidityPool.tests.ts (hunk 2 above)
Comment (ankurdubey521) [reply test/LiquidityPool.tests.ts:340]: For the sendFundsToUser call on line 371

File: test/LiquidityPool.tests.ts (hunk 3 above)
Comment (ankurdubey521) [reply test/LiquidityPool.tests.ts:371]: For the sendFundsToUser call on line 371

File: contracts/hyphen/interfaces/ILPToken.sol (hunk 1 above)
Comment (ankurdubey521) [reply contracts/hyphen/interfaces/ILPToken.sol]: fixed

File: test/LiquidityPool.tests.ts (hunk 2 above)
Comment (ankurdubey521) [reply test/LiquidityPool.tests.ts:340]: I removed the addExecutor call in beforeEach, since other tests were calling the addExecutor manually

**Review by tomarsachin2271 — APPROVED** [review]
